### PR TITLE
Enabled test cases for the port

### PR DESCRIPTION
**Pull request by HarithaIBM**
The only way to check the log is to cehck for hashes generated on running xxhsum

### CODE REVIEW

**Comment by HarithaIBM** [comment]
@IgorTodorovskiIBM  Can you please review the changes too! Am unable to add as you as reviewer

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM i invited you as a collaobrator - please accept

File: buildenv [hunk 1]
```diff
@@ -3,19 +3,38 @@
...
 export ZOPEN_INSTALL_OPTS="install POSIX_ENV=Yes prefix=\${ZOPEN_INSTALL_DIR}" 
 export ZOPEN_NAME="xxHash-0.8.1"
-export ZOPEN_CHECK="skip"
 rm -f v0.8.1
 ln -s ${ZOPEN_NAME} v0.8.1
 
 zopen_check_results()
 {
-  return 3 # non-functional
+  FAILURE=0
```
Comment (MikeFultonDev) [inline buildenv:19]: Can you add a few lines explaining what your check is doing? I'm assuming it's validating that the checksums are right but a few words would be good here.

**Review by MikeFultonDev — CHANGES_REQUESTED** [review]

File: buildenv (hunk 1 above)
Comment (HarithaIBM) [reply buildenv:19]: Sure , Mike! I have added comments. Hope that helps. Thank you!

**Comment by HarithaIBM** [comment]
This is no longer needed as the test cases are enabled in xxhashport , so closing this.

**Pull request closed without merging**